ktor-client/ktor-client-features/ktor-client-json/common/src/io/ktor/client/features/json/JsonFeature.kt

```diff
@@ -57,7 +57,10 @@ class JsonFeature internal constructor(
          *
          * Default value for [serializer] is [defaultSerializer].
          */
-        var serializer: JsonSerializer? = null
+        var serializer: JsonSerializer
```

Comment:
Please consider using `lazy` here

Reply:
Or we can move the default in the constructor parameters

Reply:
Yeah, that solution wasn't ideal. I'd like to keep the lazy behavior, but `lazy` doesn't support `var`, so I'll add a `lazyVar` to ktor-utils.